pages/common/npx.md

```diff
@@ -0,0 +1,15 @@
+# npx
+
+> Execute binaries from `npm` packages(most of the cli tools) .
```

Comment:
What is the use of "(most of the cli tools)" here ?

Reply:
to explain the packages, usually the packages is cli tools to excute like webpack